api/proto/info/info.proto

```diff
@@ -566,6 +624,20 @@ message ZInfoVpn {
   repeated ZInfoVpnConn conn = 10; // Connection Information
 }
 
+enum ZNetworkInstanceRunState {
+    ZNETINST_RUN_STATE_UNSPECIFIED = 0;
+    ZNETINST_RUN_STATE_INIT = 1;
+    // Terminal State.
+    ZNETINST_RUN_STATE_ONLINE = 2;
+    // Terminal State.
+    ZNETINST_RUN_STATE_ERROR = 3;
+    // De-Activation in Progress.
+    ZNETINST_RUN_STATE_HALTING = 4;
```

Comment:
I don't think we need a transient HALTING state here.

Reply:
Removed